.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -81,47 +81,39 @@ assignees: ''
 
 ### Relevant Database Driver(s)
 
-<!--
-  Please include the relevant database drivers to your issue.
-
-  Please include an `x` inside the brackets like so:
-
-  - [ ] `cockroachdb`
-  - [x] `cordova`
-  - [ ] `expo`
--->
-
-- [ ] `aurora-data-api`
-- [ ] `aurora-data-api-pg`
-- [ ] `better-sqlite3`
-- [ ] `cockroachdb`
-- [ ] `cordova`
-- [ ] `expo`
-- [ ] `mongodb`
-- [ ] `mysql`
-- [ ] `nativescript`
-- [ ] `oracle`
-- [ ] `postgres`
-- [ ] `react-native`
-- [ ] `sap`
-- [ ] `sqlite`
-- [ ] `sqlite-abstract`
-- [ ] `sqljs`
-- [ ] `sqlserver`
+<!-- Copy/paste "✅" to your issue's relevant databases, left "✖️" others: -->
+
+- ✅ all databases
```

Comment:
To make it easier to write, perhaps

`:x:` / `:heavy_check_mark:`?

Reply:
is the comment enough?

Reply:
(check it out, updated it above)